front-end/src/components/PostListItem.tsx

```diff
@@ -66,6 +70,33 @@ export default function PostListItem(props: Props) {
       })
   }
 
+  //function to reshare 
+  function reshare(post: Post): void{
+      //remove author id 
+      let data = {
+        title: post.title,
+        description: post.description,
+        visibility: post.visibility,
+        unlisted: post.unlisted,
+        contentType: post.contentType,
+        content: post.content,
+        categories: post.categories
+      }
+      AxiosWrapper.post(process.env.REACT_APP_API_URL + "/api/author/" + props.loggedInUser?.authorId + "/posts/", data, props.loggedInUser)
+        .then((res: any) => {
+            if (res.status >= 400) {
+                setShowError(true)
+            } else if (res.status === 201) {
+                //props.history.push("/")
```

Comment:
We are taking care of sending the posts to followers inboxes on the frontend, not the backend so you haven't shared the posts to followers inboxes yet. Here with res.data is the post with it you should send to followers inboxes. You can take a look at how its done by going to the CreateAndEditPostModal file. In there, you can take that code and put it here and it should work. We can maybe discuss later about if we want to be able to customize whether or not its sent to friends vs followers but for now, I think we can just keep it to all followers for the deadline.